R/save_e61.R

```diff
@@ -69,8 +70,8 @@
 save_e61 <- function(filename,
                      ...,
                      plot = last_plot(),
-                     format = c("svg", "pdf", "eps", "png"),
-                     chart_type = c("MN", "RN", "PPT"),
+                     format = c("svg", "pdf", "eps", "png", "jpg"),
+                     chart_type = NULL,
```

Comment:
We should enforce the `chart_type` argument to be one of the 3 options. Also would strongly prefer lowercase arguments -- `c("normal", "wide", "square")`.